src/main/webapp/search.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Capstone MVP</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body onload="determineLogin()" id= "background">
+
+<nav id="navbar">
+   <a href="index.html" class="button">Home</a>
+   <a href="user.html" class="button">Profile</a>
+   <a href="store.html" class="button">Explore</a> 
+   <a href="search.html" id="active" class="button">Search for a drink!</a> 
+</nav>
+<nav id="loginbar">
+      <div id="form"></div>
+      <div id="login-container"></div>
+      <div id="EMAIL"></div>
```

Comment:
lowercase?

Reply:
yes, i'm unsure why but I have changed email to lowercase weeks ago and this will be the third time i will do it again, but it keeps showing up as uppercase.